### PR TITLE
docs(components): fix quick start instructions

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/components/README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ The following snippet fully renders a Vivid button component
 <link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/tokens/theme-light.css">
 
 <!-- import typography for desktop -->
-<link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/typography/desktop.css">
+<link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/typography.css">
```
Comment (yinonov) [inline libs/components/README.md:134]:
```suggestion
<link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/core/typography.css">
```

File: libs/components/README.md [hunk 2]
```diff
@@ -131,10 +131,12 @@ The following snippet fully renders a Vivid button component
...
 
 <!-- import typography for desktop -->
-<link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/typography/desktop.css">
+<link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/core/typography.css">
 
 <!-- import Vivid button component -->
 <script type="module" src="https://unpkg.com/@vonage/vivid@next/button/index.js"></script>
 
-<vwc-button label="Click me" appearance="filled" connotation="cta"></vwc-button>
+<div class="vvd-root">
```
Comment (YonatanKra) [inline libs/components/README.md:139]:
```suggestion
<!-- Part of the app (or a whole app) that contains vivid components -->
<div class="vvd-root">
```

File: libs/components/README.md [hunk 3]
```diff
@@ -131,10 +135,12 @@ The following snippet fully renders a Vivid button component
 <link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/tokens/theme-light.css">
 
 <!-- import typography for desktop -->
-<link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/typography/desktop.css">
+<link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/core/typography.css">
```
Comment (YonatanKra) [inline libs/components/README.md:138]: What about `theme.css`? How and when one would use it?

Comment (yinonov) [reply libs/components/README.md]: actually, they should always use it. it has such a little effect of setting the correct `color-scheme`. this line is anyway irrelevant for the quick start. however, in ur codepen u should import all `
<link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/core/all.css">`

**Pull request merged**